docs/user-guide.rst

```diff
@@ -491,7 +491,7 @@ Reporting Issues
 
 You can report problems on our `issue tracker`_. The `source code`_ is also
 available on GitHub. The development team hangs out in ``#fedora-apps`` on the
-freenode network. Please do stop by and say hello.
+libera network. Please do stop by and say hello.
```

Comment:
It will be probably better to use libera.chat, because libera is showing multiple unrelated search result. And even better would be to have a link to https://libera.chat/